.github/workflows/msal-angular-e2e.yml

```diff
@@ -37,7 +37,6 @@ jobs:
         sample:
           - 'angular9-v2-sample-app'
           - 'angular10-sample-app'
-          - 'angular11-b2c-sample'
```

Comment:
Does the b2c sample app need to be added here too?

Reply:
working on the e2e tests separately. Will add this back afterwards